lib/github/templates/security-pos-release.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
So, this won't work. We normally update the document (pre security release blog) to inform it is ready. 

We should pull from the current pre-release blog and update on top of that.

Reply:
instead of overwriting it should append at the beginning of the existing file?

Reply:
Exactly. We also update the date and the banner of the website.

Reply:
https://github.com/nodejs/nodejs.org/pull/6342